.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I've never used yarnrc. Is it recommended when freezing a version to include the frozen version as part of your repo? It kind of looks like that based on the [yarnrc](https://legacy.yarnpkg.com/en/docs/yarnrc/) docs.

Reply:
> I've never used yarnrc. Is it recommended when freezing a version to include the frozen version as part of your repo?

I think so. This is what yarn's maintainer suggested: 

> If you don't want to upgrade all of your projects, just run yarn policies set-version ^1 in the repositories that need to stay on Yarn 1, and commit the result. Yarn will always prefer the checked-in binaries over the global ones, making it the best way to ensure that everyone in your team shares the exact same release!

I just did that, ran the command, committed the result :)

see their article https://dev.to/arcanis/introducing-yarn-2-4eh1